app/views/va_facilities/_facility_results_table_header.html.erb

```diff
@@ -0,0 +1,88 @@
+<thead>
+<tr>
+  <th scope="col"><a class="fake-link" id="toggle_by_facility">Facility
+    <% if params[:asc].blank? || params[:sortby] == "facility" %>
+      <% if params[:asc] == "true" %>
+        &#9650;
+      <% end %>
+      <% if params[:asc] == "false" %>
+        &#x25BC;
+      <% end %>
+      <% if params[:asc].blank? %>
```

Comment:
Can we use the font awesome icon here instead of html code? The down arrow can be found here:
https://www.fontawesome.com/icons/sort-down?style=solid

Also, since we are using the same icon for two separate conditions, can we just combine them, like this:
```
<% if params[:asc] == "true" || params[:asc].blank? %>
```